distribution/lib/Standard/Base/0.0.0-dev/src/Error/Common.enso

```diff
@@ -231,8 +237,7 @@ type Caught_Panic
          the source of this panic. Only for internal use. To get the Java exception
          from polyglot exceptions, match the `payload` on `Polyglot_Error` and
          extract the Java object from there.
-    @Builtin_Type
-    type Caught_Panic payload internal_original_exception
+    Caught_Panic_Data payload internal_original_exception
```

Comment:
I'm thinking if with the new machinery it wouldn't make more sense to just make `Caught_Panic` a "completely" builtin type, to avoid the `Caught_Panic_Data` constructor and just match it as `Caught_Panic` (like we do with matching `Text` or `Integer`).

Not for this PR though.

Reply:
Yeah I don't see why not just expose these directly tbh